manifests/m/Microsoft/OpenJDK/11/11.0.20.101/Microsoft.OpenJDK.11.locale.en-US.yaml

```
@@ -16,9 +16,10 @@ LicenseUrl: https://openjdk.java.net/legal/gplv2+ce.html
 ShortDescription: The Microsoft Build of OpenJDK is a new no-cost long-term supported distribution and Microsoft's new way to collaborate and contribute to the Java ecosystem.
 Moniker: openjdk-11
 Tags:
-- "11"
+- 11
```

```suggestion
- "11"
```
Related to 
- https://github.com/microsoft/winget-pkgs/issues/113794

Also it would be preferred if you reset the changes made to ManifestVersion / schema version headers. Looks like these changes are added because you're using an older version of winget-create. If you upgrade to the latest, then it'll use the latest schema and will quote these number strings properly as well.